polldaddy.php

```diff
@@ -782,34 +782,40 @@ function management_page_load() {
 					$this->errors->add( 'answer', __( 'Invalid answers', 'polldaddy' ) );
 
 				$answers = array();
-				foreach ( $_POST['answer'] as $answer_id => $answer ) {
-					$answer = stripslashes( trim( $answer ) );
+				if ( ! empty( $_POST['answer'] ) ) {
```

Comment:
And I noticed that on the `edit-poll` we have a validation to the minimum 2 answers. I think we could add the same validation to the `create-poll`. Otherwise we can create a poll with one option and it saves the poll, but without options.

Now the `create-poll` is validating if we have at least one answer. And this validation has no message. Maybe we could change to validate at least 2 answers, like the `edit-poll` and add the same message.
`$this->errors->add( 'answer', __( 'You must include at least 2 answers', 'polldaddy' ) );`

Reply:
Great testing 👍 🎉 Added in 91a2bcefa8855f6798c3197fa7a01bd0f3d8fe84